src/platforms/kotlin/config.yml

```diff
@@ -0,0 +1,10 @@
+title: Kotlin
+caseStyle: camelCase
+supportLevel: production
+sdk: sentry.kotlin
```

Comment:
this SDK does not exist, its pretty much the Java SDK, should we remove it or add the Java one?

Reply:
Will simply point to Java